data_upload_DHIS2.py

```diff
@@ -0,0 +1,74 @@
+import requests
+import json
+
+dhis2_username = ''
+dhis2_password = ''
+DHIS2_Test_Server_URL = 'https://ocr.twoca.org/'
+
+
+# Command to get all fields that are organisationUnits
+# TestServerURL/api/organisationUnits.json?fields=:all&includeChildren=true&paging=false
+
+# Get UIDS for dataSet, period, orgUnit
+# Known Words in dataSet
+def getUID(item_type, search_items):
+    filter_param = 'filter=' + '&filter='.join([f'name:ilike:{term}' for term in search_items])
+
+    url = f'{DHIS2_Test_Server_URL}/api/{item_type}?{filter_param}'
+
+    response = requests.get(url, auth=(dhis2_username, dhis2_password))
+    response.raise_for_status()
+    data = response.json()
+
+    items = data[item_type]
+    print(f"{len(data[item_type])} matches found for {search_items}")
+
+    id = items[0]['id']
```

Comment:
This function performs a fuzzy match based on the search terms provided by the user. For example, I searche for 'general OPD' and receive 50 matching results, the function may return the first match from a list of similar results. 

To ensure the ID is unique and accurate, it is recommended that the user provides the full and exact name. So I'm thinking if the search terms are incomplete, the user is prompted to refine the search or select the correct item from a list. Does that make sense?

Reply:
Yeah, definitely! With the OCR, I'm thinking that the pipeline would be OCR recognizes text, the user confirms the text is correct before searching (in case it is very wrong), the API searches for the ID, then prompts the user with a dropdown if it is not an exact match. Is that what you were thinking?

Reply:
Yeah, this is exactly what I'm thinking~